service/project.go

```diff
@@ -467,7 +467,10 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 	projectAliases = append(projectAliases, responseRef.CommitQueueAliases...)
 	projectAliases = append(projectAliases, responseRef.PatchAliases...)
 	for i := range projectAliases {
-		projectAliases[i].ProjectID = id
+		if projectAliases[i].ProjectID != id { // new project, so we need a new document (new ID)
```

Comment:
Is there a test that you can write for this? I realize this file has no coverage at all (https://s3.amazonaws.com/mciuploads/evergreen/evergreen_coverage_coverage_659180eb4df576582b1b6340ba7db9db78196a17_19_05_21_14_39_40/output.service.coverage.html#file29), and this function is crazy long. Would it be possible, for example, to refactor this for loop into a separate function, and then test that?

Reply:
Definitely refactor sounds better than testing the crazy long function haha, how's this?